src/card.css

```diff
@@ -0,0 +1,87 @@
+.card {
+  font-family: Roboto, Helvetica, sans-serif;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  max-width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-bottom: 16px;
+}
+
+.card__photo {
+  padding: 32px 19px 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  padding: 0 16px;

```

Comment:
You don't need to add horizontal padding to each element. Just add paddings to the card.